### PR TITLE
Update Node quickstart and tutorial to use grpc-js

### LINKED ISSUE #201: Adapt Node quick start and tutorial to use gRPC-JS v1.0?
@murgatroid99, @nicolasnoble: Now that [v1.0 of gRPC-JS has been released](https://grpc.io/blog/grpc-js-1.0/) and [Node grpc library](https://www.npmjs.com/package/grpc) has been deprecated, should the [Node quick start][qs] and [Node tutorial](https://grpc.io/docs/tutorials/basic/node) be updated to make use of [gRPC-JS](https://www.npmjs.com/package/@grpc/grpc-js) instead of the [Node grpc library](https://www.npmjs.com/package/grpc)?

cc @thisisnotapril 

[qs]: https://grpc.io/docs/quickstart/node

### PR DESCRIPTION
**Pull request by murgatroid99**
This fixes #201. The corresponding changes to the example code are in grpc/grpc#24679. Since these guides instruct the user to check out the release version, the instructions here will be out of sync with that code until the other PR is merged and another release is tagged.

@chalin

### CODE REVIEW

**Comment by chalin** [comment]
Thanks for the update to the Quick start and tutorial!

The changes look good, but as you point out, we'll need to wait for the next release of grpc/grpc (so I've marked this as blocked).

**Comment by chalin** [comment]
Strangely the [preview of, say, the Quick start](https://deploy-preview-500--grpc-io.netlify.app/docs/languages/node/quickstart/) refers to `v1.30.0` of grpc/grpc. Could it be that you need to rebase your branch?

**Comment by murgatroid99** [comment]
I assume it will have the right value once the PR is merged.

**Comment by chalin** [comment]
> I assume it will have the right value once the PR is merged.

Yes it will.

**Comment by chalin** [comment]
Awaiting release > v1.33.2 of [grpc/grpc](https://github.com/grpc/grpc/releases).

**Comment by chalin** [comment]
From https://github.com/grpc/grpc/pull/24679#issuecomment-729713773:

> `@grpc/grpc-js` examples should be in the [grpc/grpc-node](https://github.com/grpc/grpc-node) repo, don't you agree?
> 
> Here's my understanding of the current situation, please correct me if I'm wrong:
> 
> * The [grpc package](https://www.npmjs.com/package/grpc) is _deprecated_, it hasn't reached end-of-life.
> * This PR changes the only `grpc` example sources we have.
> 
> Wouldn't it be preferable to revert this PR and to create `@grpc/grpc-js` examples under in it's home repo?

Thoughts @murgatroid99 @jtattermusch @nicolasnoble?

**Comment by chalin** [comment]
> Awaiting release > v1.33.2 of [grpc/grpc](https://github.com/grpc/grpc/releases).

The latest [release](https://github.com/grpc/grpc/releases) is v1.34.1, so this can be merged now?
Have you given any thought to https://github.com/grpc/grpc.io/pull/500#issuecomment-733024179?

**Review by chalin — APPROVED** [review]
Since v1.34.1 is out, I'll go ahead and merge this. We can revisit later if needed.

**Pull request merged**
